packages/gatsby-image/src/index.js

```diff
@@ -13,6 +13,13 @@ const convertProps = props => {
     delete convertedProps.sizes
   }
 
+  if (convertedProps.fixedImages) {
```

Comment:
This was to avoid refactoring the existing places where we check for or use `fluid` / `fixed`, namely when deciding what to render, and in for the image cache.